### ISSUE TITLE
[RFC] Creation recipes for rocket schematics

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/GT-New-Horizons-Modpack#7374]
#### Which modpack version are you using?
2.0.1

At the moment, to progress in rocket tiers you need to find and kill a boss on each tier of planets. There's 7 of these things total, and they don't really distinguish themselves mechanically in good ways. In my humble opinion, I think it would be a good idea to give an option to fabricate these objects within your base. Normally I'm all for new and difficult tasks, however I find that for the most part, finding the dungeon, and killing the bosses is neither new, nor difficult. You have crazy speed options, basically full resistance to damage, quite high damage outputting option, and a few other tips to make this process less awful. I wouldn't say it's terribly difficult though, and most fall short of even what we see in the TF.

I would suggest the fabrication route to be semi awful to setup however. I think something that follows this general setup could be good. 

Mk2 blueprint is either still found the normal way to force a single dungeon run, or produced by scanning a piece of dungeon brick, or other moon unique item. Printing the data stick would then result in a renewable way of producing the MK2 blueprints without needing to fight mobs, or fly across planets. 

Mk2 blueprint -> rough mk3 blueprint (scanner)
Rough Mk3 + some tier appropriate stuff (maybe lasering it, or even printing maybe?) as well as something unique to the T2 planets (maybe mars sludge or something?) -> mk3 blueprint

then Mk3 -> rough Mk4 (scanner)
Rough Mk4 + ~IV processes of printing or scanning, or whatever

etc...

One issue would be what to use to give it some sense in producing a blueprint. Pre TT we don't seem to have much in the way of "processing" or "computational" power. Scanners could help, but apart from those we might need to get a little creative with it. I wouldn't be opposed to implementing more magical progression into this to account for the deficit of related machines. The think tank could possibly be used, or making it a thaum research itself possibly. Mk2 blueprint + mars stone, in an infusion altar, could be an interesting way to tackle this. Assuming you produce all of them, and they're recursive, that gives some good infusion altar automation incentive. 

I think the recursion here will make the setup something that won't ever gate your rocket progress as long as you have a dedicated setup for it. However trying to avoid automation and producing these by hand will feel tedious and unrewarding compared to the boss fights.

Basically I think it should be a reward for people who setup some infra around it, as opposed to a simple bypass mechanic. While the mobs aren't great, I would argue that they're better than needing a tiered machine and normally used parts.

**Comment by AtomicGrog** [comment]
Honestly each tier is a one off encounter and unlikely to be done in quick succession. I've actually enjoyed the boss fights albeit as you mention they are a little uncreative.

Considering how quickly it can be done (i.e. bosses can be killed) when you have access to the planet is there any real benefit apart from fight avoidance or is this more a method to allow rockets to be unlocked out of cycle

**Comment by 0lafe** [comment]
Just a way to avoid the boss fights. Ideally it would be linked to something unique to each planet tier, so no tier skip options. I personally have gone through the planets a few times now, and I don't see a ton of differentiation between the bosses. True it's not usually back to back, but I would personally like a way of avoiding them with some other cost.

Not suggesting changing the unlock order or removing the bosses, just a more costly, less world explorey option

**Comment by Glease** [comment]
There are usually some planet unique material (e.g. meteroic iron, desh, ledox etc) that you'd use to build the next tier of rocket. IMO using it in the process of making next tier schematic sounds good both gameplay wise and lore wise.

**Comment by GTNH-Colen** [comment]
I'm sure we'll disagree on the actual recipe itself but overall I agree with this idea, the bosses are a repetitive and quite frankly boring mechanic.

**Comment by 0lafe** [comment]
i think this is still relevant? i'm not sure it got added

**Comment by glowredman** [comment]
It is still relevant and it didn't get added yet. I didn't forget, I just don't have the time at the moment.
